js/bitmart.js

```diff
@@ -2771,6 +2771,88 @@ module.exports = class bitmart extends Exchange {
         };
     }
 
+    async transfer (code, amount, fromAccount, toAccount, params = {}) {
+        /**
+         * @method
+         * @name bitmart#transfer
+         * @description transfer currency internally between wallets on the same account
```

Comment:
Can we add to the description that currently, we can only transfer between spot and margin?